src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -388,23 +384,19 @@ private DependencyBehavior GetDependencyBehavior()
                             Console)
                     };
 
+                    if (EffectivePackageSaveMode != Packaging.PackageSaveMode.None)
+                    {
+                        projectContext.PackageExtractionContext.PackageSaveMode = EffectivePackageSaveMode;
```

Comment:
how is this changed `projectContext` flowing into `downloadContext` like the previous code?

Reply:
https://github.com/NuGet/NuGet.Client/pull/2428/files#diff-c9a788561af1164b685a274237b25c53L375